docs/pages/connect-your-client/putty.mdx

```diff
@@ -0,0 +1,223 @@
+
+For example, if your leaf cluster is named `example.teleport.sh` and your node is called `ip-172-31-34-128.us-east-2.compute.internal`,
+you can add a PuTTY session for the login `ec2-user` using the following command:
+
+```bash
+C:\Users\gus>tsh puttyconfig --leaf example.teleport.sh ec2-user@ip-172-31-34-128.us-east-2.compute.internal
+Added PuTTY session for ec2-user@ip-172-31-34-128.us-east-2.compute.internal [leaf:webby.teleport.sh,proxy:teleport.example.com]
+```
+
+
+## Session naming
+
+Sessions are named using the following schema:
+
+Root clusters: `<login>@<hostname> [proxy:<proxy address>]`
+
+Leaf clusters: `<login>@<hostname> [leaf:<leaf cluster name>,proxy:<proxy address>]`
+
+## Frequently asked questions
+
+### Do I need administrator rights on my machine to run `tsh puttyconfig`?
+
+No, `tsh` only makes changes to the `HKEY_CURRENT_USER` registry key.
+
+### Can I add saved PuTTY sessions for all users on a machine?
+
+No, sessions are only added for the current user. You can export the sessions for another user to import using the
+registry export method described below.
+
+### Can I export my saved PuTTY sessions to another machine?
+
+Yes, use the Registry Editor to export the `HKEY_CURRENT_USER\Software\SimonTatham\PuTTY` registry key to a file
+and import this file on another machine. Note that you will need admin rights to run Registry Editor.
+
+### Can I change the font size, window size or other preference for my saved PuTTY sessions?
+
+Once a session has been added, you can make any desired modifications in the PuTTY UI by clicking the session name in the
+"Saved sessions" list, then clicking "Load". Make all the necessary changes, then choose the session again and click "Save".
+
+### If I re-run `tsh puttyconfig` for a given host, will it overwrite any custom changes I've made to the saved session?
+
+Teleport will only modify the config parameters that it relies on, like proxy name, proxy command, hostname, username etc.
+Any changes to font size, window size and other parameters will be untouched.
+
+### Can I use other graphical clients like MobaXterm or SecureCRT?
+
+Only PuTTY support is implemented at present. Most other graphical clients do not fully support the use of both SSH user
+certificates and SSH host certificates for authentication which is a requirement to use Teleport.
+
+Please contact the authors of those clients if you wish to see support for Teleport sessions added.
+
+### Can I use forks of PuTTY like KiTTY or Solar PuTTY?
+
+If the fork uses PuTTY 0.78 or higher as its base, it may work. However, the Teleport team only tests stock versions of PuTTY
+and cannot provide support for forked versions.
+
+### Can I use my saved Teleport PuTTY sessions with WinSCP?
+
+At the time of writing, WinSCP has not implemented support for validation of host certificates, which means WinSCP is unable
+to connect via Teleport.
```

Comment:
```suggestion
At the time of writing, WinSCP doesn't support validation using host certificates. Because Teleport requires host certificate validation, you can't use WinSCP to connect to a host through Teleport.
```